devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -91,6 +91,16 @@ def parse(String description) {
             event.value = event.value.toLowerCase()
         }
         sendEvent(event)
+    }else if(description?.startsWith('zone status') || description?.startsWith('zone report')){
+        def splitMsg = description.split(" ");
```

Comment:
We have a helper function for parsing the Zone Status attribute that you should use instead:

```zs = zigbee.parseZoneStatus(description)
value = zs.isBatterSet() ? 5 : 50
```